tests/tensor/rewriting/test_basic.py

```diff
@@ -1298,14 +1297,16 @@ def test_local_join_make_vector():
 
 
 def test_local_sum_make_vector():
+    # Check that rewrite is applied. Enforce dtype to allow rewrite
+    # even if floatX != "float64"
     a, b, c = scalars("abc")
     mv = MakeVector(config.floatX)
-    output = mv(a, b, c).sum()
+    output = mv(a, b, c).sum(dtype="float64")
 
     output = rewrite_graph(output)
     between = vars_between([a, b, c], [output])
     for var in between:
-        assert (var.owner is None) or (not isinstance(var.owner.op, MakeVector))
+        assert (var.owner is None) or (isinstance(var.owner.op, Elemwise))
```

Comment:
These tests were already hard to read before but its worse with Elemwise which could be anything. 

Could we instead use the approach `assert equal_computations([output], [expected_output])` that we use in other rewrite tests? I can link to some examples later

Reply:
I agree. I took a stab at some better tests. Let me know what you think.